docs/guide/topics/embeds.rst

```diff
@@ -0,0 +1,460 @@
+:orphan:
+.. currentmodule:: discord
+
+.. _guide_topic_embeds:
+
+
+Embeds
+=======
+
+Embeds are special message that are formatted to embed rich content within. You may have ever sent a link to a server and
+seen one appear with a brief description and maybe an image or video.
+
+.. |link_embed_google| image:: /images/guide/topics/embeds/link_embed_google.png
+   :scale: 38%
+   :target: https://google.com/search?q=discord%20inc
+.. |link_embed_discord| image:: /images/guide/topics/embeds/link_embed_discord.png
+   :scale: 38%
+   :target: https://discord.com
+.. |link_embed_youtube| image:: /images/guide/topics/embeds/link_embed_youtube.png
+   :scale: 38%
+   :target: https://www.youtube.com/watch?v=TJ13BA3-NR4&
+.. |link_embed_github| image:: /images/guide/topics/embeds/link_embed_github.png
+   :scale: 38%
+   :target: https://github.com/Rapptz/discord.py
+
++---------------------+----------------------+----------------------+---------------------+
+| |link_embed_google| | |link_embed_discord| | |link_embed_youtube| | |link_embed_github| |
++---------------------+----------------------+----------------------+---------------------+
+
+The examples above are just a few common samples of what you seen often around Discord, these are automatically generated 
+by discord based on the metadata from the linked sites but you don't need to remember any of this.
+
+Internally, an embed is represented with a JSON payload by the API. discord.py offers a 
+builder-esque interface over this payload to help make the process more straightforward and intuitive in Python.
+
+Let's take a look at a basic example of using the builder:
+
+.. code-block:: python
+
+    import discord
+
+    embed = discord.Embed(
+        title = "Hello World",
+        description = "This is a description",
+        colour = discord.Colour.blurple()
+    )
+
+And then we can send it to a channel using the ``embed`` keyword-only argument:
+
+.. code-block:: python
+
+    await channel.send(embed=embed)
+
+On Discord, this will look like:
+
+.. image:: /images/guide/topics/embeds/basic_embed.png
+    :scale: 50%
+
+Let's break down what we did.
+
+First, we imported the ``discord`` module. This is required to access the ``Embed`` class.
+
+.. note::
+
+    You can also import the ``Embed`` class directly from the ``discord`` module like so:
+
+    .. code-block:: python
+
+        from discord import Embed
+
+        embed = Embed(...)
```

Comment:
This feels entirely unnecessary and out of place to me, and I know this guide is intended to be more simple prose but this feels like explaining Python abc building blocks.

Reply:
The guide is focused on beginners of Python kindof but ig this is too much. Will remove on the next commit(s).